package-lock.json

```diff
@@ -1,8605 +1,8688 @@
 {
```

Comment:
package.json was not changed, so I don't see why package-lock.json was changed. Consider reverting this file.